docs/intro/tutorial07.txt

```diff
@@ -174,7 +174,8 @@ tabular way of displaying inline related objects. To use it, change the
 .. code-block:: python
     :caption: ``polls/admin.py``
 
-    class ChoiceInline(admin.TabularInline): ...
```

Comment:
This is not a complete snippet, it only displays what should be changed in the previous declaration i.e, `StackedInline` to `TaburalInline`.